packages/clarity-js/types/core.d.ts

```diff
@@ -3,6 +3,8 @@ import { Metadata, Payload, Token } from "./data";
 type TaskFunction = () => Promise<void>;
 type TaskResolve = () => void;
 type UploadCallback = (data: string) => void;
+type Region = [number, string, number?, string?];
```

Comment:
Use the correct typing here. I think the second (?) is a RegionFilter and for MEtric the second is a Extract.

Reply:
Added comments and used references to specific types where possible.